src/components/RoomView/ResetTeams.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ref, remove, set } from 'firebase/database';
+import { database } from '../../utils/firebase';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../store';
+
+const ResetTeams = () => {
+    const { roomId } = useSelector((state: RootState) => state.player);
+    const { status } = useSelector((state: RootState) => state.game);
+    const teamOneSpymasterRef = ref(database, `rooms/${roomId}/team-1/spymaster/`);
+    const teamTwoSpymasterRef = ref(database, `rooms/${roomId}/team-2/spymaster/`);
+    const teamOneOperativesRef = ref(database, `rooms/${roomId}/team-1/operatives/`);
+    const teamTwoOperativesRef = ref(database, `rooms/${roomId}/team-2/operatives/`);
+    // if host, see resetTeams button
+const resetTeams = () => {
+    remove(teamOneSpymasterRef)
+    remove(teamTwoSpymasterRef)
+    remove(teamOneOperativesRef)
+    remove(teamTwoOperativesRef)
+}
+    return (
+        <>
+        {status === 'ready' && 
```

Comment:
i do think moving this conditional to be in the parent / where is is being called is cleaner, but i think for this one, where it probably doesnt matter if the component stays mounted or not, can stay here if you prefer

Reply:
i tried this but for whatever reason, roomView isn't recognizing updated game status